backends/arm/tosa_utils.py

```diff
@@ -132,3 +135,33 @@ def buildReshape(tosa_fb, input_name, new_shape, output_name):
     attr = ts.TosaSerializerAttribute()
     attr.ReshapeAttribute(new_shape)
     tosa_fb.addOperator(TosaOp.Op().RESHAPE, [input_name], [output_name], attr)
+
+
+def isPermuteNodeBeforeAddMM(node):
```

Comment:
This should follow the pep008 conventions and be lower case with underscores. same for isBiasNodeForAdMM below. (I know we're not exclusively correct in the existing code but we can move towards it).

Reply:
agree. refactored those two (isPermuteNodeBeforeAddMM and isBiasNodeForAdMM). will do another full path to refactor all the relevant code.

Reply:
Feel free to create issues and assign it to yourself :p

Reply:
yep, sounds good.